1208-projects/user_edits_project.feature

```diff
@@ -0,0 +1,93 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User with access rights for Project
```

Comment:
Hvis flere enn Project Manager skal kunne endre prosjekt bør `Project Manager opens Landing Page for Project` evt utvides til å dekke det også
```suggestion
        Given Project Manager opens Landing Page for Project
```

Reply:
Har skrevet om senario